### ISSUE TITLE
Not correctly firing the ecommerce bits on the [give_receipt] page, potentially others.

### ISSUE THREAD

**Issue by Benunc** [WordImpress/Give-Google-Analytics#17]
https://youtu.be/Yni2PSfJI7Q

Also, other reports of similar (Topher DeRosia reached out via slack with a client having similar issues, also using Monster Insights).

**Comment by Benunc** [comment]
Also, I don't think there is anything at all firing on the hook here:

`add_action( 'give_payment_receipt_after_table', 'give_google_analytics_completed_donation', 10, 2 );`

**Comment by Benunc** [comment]
possibly related: https://www.kathirvel.com/yoast-google-analytics-reassign-gatracker-to-ga/

**Comment by Benunc** [comment]
user followed up here today.

**Comment by Benunc** [comment]
Followed up again today on this. This issue is a blocker for the main functionality of the addon.

**Comment by jaydeeprami** [comment]
@DevinWalker, I have setup Google analytics and trying to reproduce this issue as @Benunc mentioned in Video. I found that, Revenue generate properly but not passed other information in it. Also, I don't get any JS error as per @Benunc Video in issue.

See video of my testing:
https://screencast-o-matic.com/watch/cFV6iLoQrz

Please let me should I workaround it?

**Comment by mathetos** [comment]
@DevinWalker this is the summary of Jaydeeps findings according to his video:

Summary:
* Event tracking is working
* The event appears in Product Performance
* The event does appear in Sales performance with amounts reflected.
* But when you go to that particular event it is not saving the Product name, Product Revenue, and Quantity correctly at all.

So there is an issue, but his is different from Ben's.

There's a couple questions remaining though between Ben's video and Jaydeep's:

1) I don't know what version of Monster Insights Ben was running on.
2) Jaydeep didn't test with Monster Insights at all it seems, so it's not clear whether Ben's issue is still valid because it might be an incompatibility with Monster Insights directly. @jaydeeprami how to you include the basic GA script? Just via the JS snippet in your theme?

Bottom line is that there's definitely issues, and it'll require some fixing for sure.

**Comment by jaydeeprami** [comment]
@mathetos Yes, I have included GA script into the theme directly. Also, I haven't Monster theme so didn't test it.

**Comment by mathetos** [comment]
@jaydeeprami @DevinWalker What's the status on this issue? We have several users still unable to use this addon at all.

**Comment by DevinWalker** [comment]
Resolved via https://github.com/WordImpress/Give-Google-Analytics/commit/32a106f03ff4aecca2a0a53322f7224a34fed2c8

Released in 1.1.2 

I'm going to be working with customers to ensure it's working as they expect now.
